include/fiction/algorithms/physical_design/post_layout_optimization.hpp

```diff
@@ -795,9 +799,11 @@ class post_layout_optimization_impl
         static_assert(is_cartesian_layout_v<Lyt>, "Lyt is not a Cartesian layout");
 
         const mockturtle::stopwatch stop{pst.time_total};
-        pst.x_size_before             = plyt.x() + 1;
-        pst.y_size_before             = plyt.y() + 1;
+        pst.x_size_before = plyt.x() + 1;
+        pst.y_size_before = plyt.y() + 1;
+
         uint64_t max_gate_relocations = ps.max_gate_relocations.value_or((plyt.x() + 1) * (plyt.y() + 1));
+        bool     optimize_pos_only    = ps.optimize_pos_only;
```

Comment:
I would drop this variable and instead refer to `ps.optimize_pos_only` in the code below